pages/lib/refinery/pages/configuration.rb

```diff
@@ -61,9 +61,10 @@ def layout_template_whitelist
     self.absolute_page_links = false
     self.types = Types.registered
     self.auto_expand_admin_tree = true
+    self.reserved_paths = %w(/rails/active_storage)
     self.friendly_id_reserved_words = %w(
       index new session login logout users refinery admin images
-    )
+    ) + self.reserved_paths.map { |path| path.split('/') }.flatten.uniq - [""]
```

Comment:
You probably just need to add `rails` because we already just add `refinery` or `admin` in `friendly_id_reserved_words ` even if there are sub path to these paths.

Reply:
I now add only the first elements from each element of reserved_paths to friendly_id_reserved_words, so in the case we have only one element '/rails/active_storage' it will only take 'rails'. If you add several paths it will remove all 'root'-fragments of the paths

Reply:
maybe we could use:

```ruby
    self.friendly_id_reserved_words = %w(
      index new session login logout users refinery admin images
    ) | reserved_paths.map { |path| path.split('/') }.flatten.uniq.reject(&:blank?)
```

Reply:
I changed it to

```
self.friendly_id_reserved_words = %w(
  index new session login logout users refinery admin images
) | self.reserved_paths.map { |path| path.split('/').reject(&:blank?).first}.flatten.uniq
```
to take only the all root_fragments of the reserved paths.